packages/guides-cli/src/Command/Run.php

```diff
@@ -76,6 +77,9 @@ protected function execute(InputInterface $input, OutputInterface $output): int
         $outputDir = $this->getAbsolutePath((string) ($input->getArgument('output') ?? ''));
         $sourceFileSystem = new Filesystem(new Local($input->getArgument('input')));
         $sourceFileSystem->addPlugin(new Finder());
+
+        $this->commandBus->handle(new LogToFileCommand($outputDir . '/logs'));
```

Comment:
I don't think this should be done via the command bus. The commandbus integration is only for parts of the library that need to be executed in a certain way. Monolog is not part of the core project of guides, we allow people to inject their own logger, So we can do this directly using monolog in the cli project. As that is an application, ready to use for end-users. 

Doing it via the command bus makes is more part of the core guides library, which I think is not a good idea. 

My suggestion is to make an input option to set the log path. which defaults to PHP://stderr.

Reply:
done, please have another look